src/UKHO.S100PermitService.StubService/Stubs/ProductKeyServiceStub.cs

```diff
@@ -8,21 +9,74 @@ namespace UKHO.S100PermitService.StubService.Stubs
 {
     public class ProductKeyServiceStub : IStub
     {
-        private readonly ProductKeyServiceConfiguration _productKeyServiceConfiguration;
```

Comment:
This is correct, should be '_productKeyServiceConfiguration'. Please amend the editor rule.

Reply:
Resolved